diff --git a/node_modules/@yarnpkg/core/lib/Configuration.js b/node_modules/@yarnpkg/core/lib/Configuration.js
index 108c033..f3ff070 100644
--- a/node_modules/@yarnpkg/core/lib/Configuration.js
+++ b/node_modules/@yarnpkg/core/lib/Configuration.js
@@ -190,7 +190,7 @@ exports.coreDefinitions = {
     enableProgressBars: {
         description: `If true, the CLI is allowed to show a progress bar for long-running events`,
         type: SettingsType.BOOLEAN,
-        default: !ci_info_1.isCI && process.stdout.isTTY && process.stdout.columns > 22,
+        default: false, //!ci_info_1.isCI && process.stdout.isTTY && process.stdout.columns > 22,
         defaultText: `<dynamic>`,
     },
     enableTimers: {
diff --git a/node_modules/@yarnpkg/core/lib/Project.js b/node_modules/@yarnpkg/core/lib/Project.js
index 345968f..90ce843 100644
--- a/node_modules/@yarnpkg/core/lib/Project.js
+++ b/node_modules/@yarnpkg/core/lib/Project.js
@@ -795,7 +795,7 @@ class Project {
         for (const locatorHash of buildablePackages)
             readyPackages.delete(locatorHash);
         const globalHashGenerator = crypto_1.createHash(`sha512`);
-        globalHashGenerator.update(process.versions.node);
+        // globalHashGenerator.update(process.versions.node);
         this.configuration.triggerHook(hooks => {
             return hooks.globalHashGeneration;
         }, this, (data) => {
diff --git a/node_modules/@yarnpkg/core/lib/StreamReport.js b/node_modules/@yarnpkg/core/lib/StreamReport.js
index 587c9a1..8b4ab6f 100644
--- a/node_modules/@yarnpkg/core/lib/StreamReport.js
+++ b/node_modules/@yarnpkg/core/lib/StreamReport.js
@@ -111,7 +111,7 @@ class StreamReport extends Report_1.Report {
             throw new Error(`Assertion failed: Invalid progress bar style`);
         this.progressStyle = PROGRESS_STYLES[styleName];
         const PAD_LEFT = `➤ YN0000: ┌ `.length;
-        const maxWidth = Math.max(0, Math.min(process.stdout.columns - PAD_LEFT, 80));
+        const maxWidth = Math.max(0, Math.min(80 - PAD_LEFT, 80));
         this.progressMaxScaledSize = Math.floor(this.progressStyle.size * maxWidth / 80);
     }
     static async start(opts, cb) {
diff --git a/node_modules/@yarnpkg/core/lib/httpUtils.js b/node_modules/@yarnpkg/core/lib/httpUtils.js
index c720ddd..f0cb0ab 100644
--- a/node_modules/@yarnpkg/core/lib/httpUtils.js
+++ b/node_modules/@yarnpkg/core/lib/httpUtils.js
@@ -65,6 +65,7 @@ function getNetworkSettings(target, opts) {
     return mergedNetworkSettings;
 }
 exports.getNetworkSettings = getNetworkSettings;
+var got_1 = tslib_1.__importStar(require('got'));
 var Method;
 (function (Method) {
     Method["GET"] = "GET";
@@ -104,7 +105,7 @@ async function request(target, body, { configuration, headers, json, jsonRequest
     const retry = configuration.get(`httpRetry`);
     const rejectUnauthorized = configuration.get(`enableStrictSsl`);
     const caFilePath = networkConfig.caFilePath;
-    const { default: got } = await Promise.resolve().then(() => tslib_1.__importStar(require(`got`)));
+    const { default: got } = got_1;
     const certificateAuthority = caFilePath
         ? await getCachedCertificate(caFilePath)
         : undefined;
diff --git a/node_modules/@yarnpkg/core/lib/scriptUtils.js b/node_modules/@yarnpkg/core/lib/scriptUtils.js
index bae40a9..ee51aae 100644
--- a/node_modules/@yarnpkg/core/lib/scriptUtils.js
+++ b/node_modules/@yarnpkg/core/lib/scriptUtils.js
@@ -5,7 +5,7 @@ const tslib_1 = require("tslib");
 const fslib_1 = require("@yarnpkg/fslib");
 const fslib_2 = require("@yarnpkg/fslib");
 const libzip_1 = require("@yarnpkg/libzip");
-const shell_1 = require("@yarnpkg/shell");
+// const shell_1 = require("@yarnpkg/shell");
 const binjumper_1 = require("binjumper");
 const capitalize_1 = tslib_1.__importDefault(require("lodash/capitalize"));
 const p_limit_1 = tslib_1.__importDefault(require("p-limit"));
