diff --git a/node_modules/memfs/lib/setImmediate.js b/node_modules/memfs/lib/setImmediate.js
index d27ef23..d9c85d4 100644
--- a/node_modules/memfs/lib/setImmediate.js
+++ b/node_modules/memfs/lib/setImmediate.js
@@ -4,5 +4,5 @@ var _setImmediate;
 if (typeof setImmediate === 'function')
     _setImmediate = setImmediate.bind(global);
 else
-    _setImmediate = setTimeout.bind(global);
+    _setImmediate = (f) => Promise.resolve().then(f);
 exports.default = _setImmediate;
diff --git a/node_modules/memfs/lib/volume.js b/node_modules/memfs/lib/volume.js
index 1af9b10..f04ecb5 100644
--- a/node_modules/memfs/lib/volume.js
+++ b/node_modules/memfs/lib/volume.js
@@ -542,7 +542,7 @@ var Volume = /** @class */ (function () {
     };
     Volume.prototype.newFdNumber = function () {
         var releasedFd = this.releasedFds.pop();
-        return typeof releasedFd === 'number' ? releasedFd : Volume.fd--;
+        return typeof releasedFd === 'number' ? releasedFd : Volume.fd++;
     };
     Volume.prototype.createNode = function (isDirectory, perm) {
         if (isDirectory === void 0) { isDirectory = false; }
@@ -1805,7 +1805,7 @@ var Volume = /** @class */ (function () {
      * @type {number}
      * @todo This should not be static, right?
      */
-    Volume.fd = 0x7fffffff;
+    Volume.fd = 10;
     return Volume;
 }());
 exports.Volume = Volume;
