diff --git a/node_modules/stream-http/index.js b/node_modules/stream-http/index.js
index 84bfe51..84c40c5 100644
--- a/node_modules/stream-http/index.js
+++ b/node_modules/stream-http/index.js
@@ -15,7 +15,7 @@ http.request = function (opts, cb) {
 	// Normally, the page is loaded from http or https, so not specifying a protocol
 	// will result in a (valid) protocol-relative url. However, this won't work if
 	// the protocol is something else, like 'file:'
-	var defaultProtocol = global.location.protocol.search(/^https?:$/) === -1 ? 'http:' : ''
+	var defaultProtocol = global.location && global.location.protocol.search(/^https?:$/) === -1 ? 'http:' : ''
 
 	var protocol = opts.protocol || defaultProtocol
 	var host = opts.hostname || opts.host
